cypress/e2e/GUI/greeting.cy.js

```diff
@@ -0,0 +1,22 @@
+describe('Teste de saudação na página inicial', () => {
+  it('Deve exibir a saudação padrão com a data atual ao acessar a página inicial', () => {
+    cy.visit('/')
+
+    const currentDate = new Date().toLocaleDateString('en-US', { weekday: 'short', month: 'short', day: '2-digit', year: 'numeric' });
+    const currentDateWithoutComma = currentDate.replace(/,/g, '');
+    const greetingWithDate = `Hi there! It is now ${currentDateWithoutComma}.`;
+
+    cy.get('[data-testid="table"]').should('contain', greetingWithDate);
+  });
+
+  it('Deve exibir a saudação customizada com a data atual ao acessar a página inicial', () => {
+    cy.visit('/')
+    cy.get('[data-testid="name"]').should('be.enabled').click().type("Guilherme Otto");
```

Comment:
Não há necessidade de clicar no campo, basta digitar no mesmo.

```suggestion
    cy.get('[data-testid="name"]').type("Guilherme Otto");
```